app/dashboard/controller.ts

```diff
@@ -117,6 +117,7 @@ export default class Dashboard extends Controller {
     @action
     more(this: Dashboard) {
         this.get('findNodes').perform(true);
+        this.analytics.click('button', 'Dashboard - load_nodes');
```

Comment:
Snuck in some analytics, 'eh? 😉

Reply:
No, I think I just moved them from the template because two onclick actions.